torch/ao/quantization/backend_config/executorch.py

```diff
@@ -47,6 +47,42 @@
     is_dynamic=True,
 )
 
+# xnnpack compatible dtype configs
+
+# We restrict scale values to be 2 ** -12 to ensure the
+# requantization scale never falls below the xnnpack lower
+# threshold. Additionally, for qint8 weight, we restrict
+# the quantization values to [-127, +127], excluding -128.
+# For more detail, refer to the description of
+# `default_symmetric_qnnpack_qconfig`.
+
+# TODO: add additional restriction on qscheme to ensure it
+# is either per_tensor_symmetric or per_channel_symmetric
+
+executorch_act_qint8_scale_min_2_neg_12 = DTypeWithConstraints(
```

Comment:
I think you are missing import here? Also idk if related but DTypeWithConstraints was giving me errors with `test_public_bindings`, i had to add it to the `__all__` in `backend_config/__init__.py` to get it to go away.

Reply:
I see, makes sense, will update